packages/expo-updates/android/build.gradle

```diff
@@ -6,17 +6,7 @@ apply plugin: 'maven-publish'
 group = 'host.exp.exponent'
 version = '0.15.4'
 
-def ex_updates_native_debug_env = System.getenv("EX_UPDATES_NATIVE_DEBUG") ?: "0"
-def ex_updates_native_debug = ex_updates_native_debug_env == "1" ? "true" : "false"
-
-// For native updates debugging, override these properties
-if (findProject(":app")) {
-  def appProjectExt = project(":app").ext
-  if (appProjectExt.has("react") && ex_updates_native_debug) {
-    appProjectExt.react.bundleInDebug = true
-    appProjectExt.react.devDisabledInDebug = true
-  }
-}
+def ex_updates_native_debug = System.getenv("EX_UPDATES_NATIVE_DEBUG") == "1" ? "true" : "false"
```

Comment:
[expo-updates] move the EX_UPDATES_NATIVE_DEBUG bundleInDebug setup to templates. unfortunately, because RNGP setups the task dependencies pretty early. the original setup in expo-updates build.gradle is too late. note that the change doesn't reference any files from templates to expo-updates package. somehow it's not tightly coupled.
cc @douglowder